src/app/View.elm

```diff
@@ -34,8 +41,8 @@ firstPlayer players =
             "No Players Found"
 
 
-tableView : List Player -> Dice.Cup -> Html Msg
-tableView players cup =
+tableView : Int -> List Player -> Dice.Cup -> Html Msg
+tableView currentPlayerIndex players cup =
```

Comment:
We need some way to tell the `tableView` which of the players is currently having their turn. Unsure if the backend allows for this.